### PR TITLE
Unit test for pretrained checkpoints

### PR DESCRIPTION
**Pull request by gaozhihan**
Run `pytest -q ROOT_DIR/tests/unittests/test_pretrained_checkpoints.py` to verify the correctness of loading pretrained checkpoints.

### CODE REVIEW

**Comment by sxjscience** [comment]
Shall we close https://github.com/amazon-science/earth-forecasting-transformer/pull/14?

**Comment by gaozhihan** [comment]
> Shall we close #14?

Done

File: scripts/cuboid_transformer/earthnet_w_meso/cfg_pretrained.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+dataset:
```
Comment (sxjscience) [inline scripts/cuboid_transformer/earthnet_w_meso/cfg_pretrained.yaml:1]: Can we rename the `cfg_pretrained.yml`? May be  call it `earthformer_enso_v1.yaml`?

Comment (gaozhihan) [reply scripts/cuboid_transformer/earthnet_w_meso/cfg_pretrained.yaml]: > Can we rename the `cfg_pretrained.yml`? May be call it `earthformer_earthnet_v1.yaml`?

Done. They are renamed as [`earthformer_sevir_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/sevir/earthformer_sevir_v1.yaml), [`earthformer_enso_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/enso/earthformer_enso_v1.yaml) and [`earthformer_earthnet_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/earthnet_w_meso/earthformer_earthnet_v1.yaml) respectively.

**Pull request merged**